frontend/src/App.vue

```diff
@@ -165,7 +164,7 @@ const onPageLoad = async () => {
     resolution: deviceRes,
     effective_resolution: effectiveDeviceRes,
     user_agent: navigator.userAgent,
-    locale: defaultLocale(),
+    locale: defaultLocale(), // needs to import defaultLocale from utils
```

Comment:
I'm a bit confused by how this still works when the `import defaultLocale` has been removed and is required as noted by the comment.

Reply:
Haha well, that whole part is commented out. Which is why the linter is complaining in the first place. We had unused imports because of this commented out code. I removed the imports now and left a comment inside the commented area to import it again should we need that code in the future. We should ask @MelissaAutumn if we could delete that commented code altogether.

Sorry for the confusion 😅

Reply:
Ahhh hahhaha my bad I didn't even notice that entire section was commented out. Thanks 😃